app/src/main/java/com/nutomic/syncthingandroid/activities/MainActivity.java

```diff
@@ -258,20 +260,27 @@ public void onCreate(Bundle savedInstanceState) {
         // SyncthingService needs to be started from this activity as the user
         // can directly launch this activity from the recent activity switcher.
         Intent serviceIntent = new Intent(this, SyncthingService.class);
+        if (getIntent().getBooleanExtra(MainActivity.EXTRA_RESET_DATABASE, false)) {
+            serviceIntent.setAction(SyncthingService.ACTION_RESET_DATABASE);
+        }
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
             startForegroundService(serviceIntent);
         } else {
             startService(serviceIntent);
         }
 
+        // In case start_into_web_gui option is enabled, immediately open that.
+        if (mPreferences.getBoolean(Constants.PREF_START_INTO_WEB_GUI, false)) {
+            startActivity(new Intent(this, WebGuiActivity.class));
+        }
```

Comment:
Is this also a bug fix?

Reply:
No, moved from first start wizard. Which actually is not only unnecessary, but worse: This should only happen when newly opening the app, not when e.g. coming back to it through the android app switcher.